README.rst

```diff
@@ -1,132 +1,200 @@
-You might also want to visit `our website <http://coala-analyzer.org/>`_.
+.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+
+coala: Language Independent Code Analysis
+-----------------------------------------
+
+**coala provides a unified command-line interface for linting and fixing all
+your code, regardless of the programming languages you use.**
+
+With coala, users can create
+`rules and standards <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
+to be followed in the source
+code. coala has an **user-friendly interface** that is completely customizable.
+It can be used in any environment and is completely modular.
+
+coala has a set of official bears (plugins) for several languages, including
+popular languages such as **C/C++**, **Python**, **JavaScript**, **CSS**,
+**Java** and many more, in addition to some generic language independent
+algorithms. To learn more about the different languages supported and the
```

Comment:
maybe move the last sentence on a newline with a newline between the paragraphs to emphasize it